R/addins.R

```diff
@@ -0,0 +1,21 @@
+create_module <- function() {
+  rstudioapi::insertText(
+"box::use(
+  shiny[moduleServer, NS]
+)
+
+#' @export
+ui <- function(id) {
+  ns <- NS(id)
+
+}
+
+#' @export
+server <- function(id) {
+  moduleServer(id, function(input, output, session) {
+
+  })
+}
+"
+  )
+}
```

Comment:
Consider `glue::trim()` for readability:
```suggestion
create_module <- function() {
  rstudioapi::insertText(glue::trim("
    box::use(
      shiny[moduleServer, NS]
    )

    #' @export
    ui <- function(id) {
      ns <- NS(id)

    }

    #' @export
    server <- function(id) {
      moduleServer(id, function(input, output, session) {

      })
    }"
  ))
}
```